### ISSUE TITLE
Backups and external sd card lifespan

### ISSUE THREAD

**Issue by anaxonda** [NeoApplications/Neo-Backup#748]
Wondering if anyone has any practical experience with sd cards failing, and effect of running daily backups on external sd card lifespan.

I've had a card recently fail (400 gb sandisk), and wonder if it could be the effect of writing and deleting data from it every day with neo-backup (probably at least several gb per day, including Signal messenger backup). I'm going to try to replace it with a so called "high endurance" version, and see if that gives me better sd card life.

**Comment by MrEngineerMind** [comment]
Yes, the High Endurance will last much longer.

**Comment by hg42** [comment]
yes, when I made daily backups a few years ago, I had several damaged cards due to wear.

They always had the same failure effect:
They silently became write protected.

This felt very mysterious, because everything seemed to work, but when rebooting the device the sdcard contents jumped back to a former state.

The reason is caching...
the system doesn't know it's write protected, but serves all the changes from the cache anyways.

If you write too much data for caching, it also jumps back from time to time, when the data has to be read again.

To solve this, I changed my backup strategy from saving everything every day to a more sophisticated scheme.

Of my over 600 apps, the biggest part isn't important at all or it does not create important data.
First I excluded most system apps... (~ 200 apps less).
Today my backup set is ~150 apps, and I have groups of apps with different backup frequency from a few days to weeks.
